File: Data-Structures-and-Algorithms/0026-RemoveDuplicatesfromSortedArray/removeDuplicates.py
class Solution(object):
    def removeDuplicates(self, nums):
        curind = 1
        for i in range(1, len(nums)):       
            if nums[i] != nums[curind-1]:
                nums[curind] = nums[i]
                curind += 1
        print(nums)        
        return (curind)

nums = [0,0,1,1,1,2,2,3,3,4]
print(Solution().removeDuplicates(nums))

# O(n)
# O(1)
# Remove repeated elements

File: Data-Structures-and-Algorithms/0026-RemoveDuplicatesfromSortedArray/removeDuplicates.js
const removeDuplicates = function(nums) {
    let insind = 1;
    for (var i = 1; i < nums.length; i++) {
        if (nums[i] != nums[insind-1]) {
            nums[insind] = nums[i];
            insind ++;
        }
    }
    return insind, nums
};

nums = [0,0,1,1,1,2,2,3,3,4]
console.log(removeDuplicates(nums))

// O(n)
// O(1)
// Remove repeated elements